docs/release-notes/8.4.asciidoc

```diff
@@ -1,6 +1,25 @@
 [[release-notes-header-8.4.0]]
 == 8.4
 
+[discrete]
+[[release-notes-8.4.2]]
+=== 8.4.2
+
+[discrete]
+[[known-issue-8.4.2]]
+==== Known issues
+* A new Lucene 9 validation change may cause EQL rule errors whenever rule queries contain regular expressions using wildcard fields and predefined character classes (for example, `\w`, `\s`, `\d`).
```

Comment:
Technically the rule is called event correlation, but maybe we can put `"may cause event correlation (EQL) rule errors...`?
```suggestion
* A new Lucene 9 validation change may cause event correlation rule errors whenever rule queries contain regular expressions using wildcard fields and predefined character classes (for example, `\w`, `\s`, `\d`).
```

Reply:
Yeah, will do. Good suggestion, thanks!